core/arch/arm/kernel/link.mk

```diff
@@ -217,15 +217,12 @@ $(link-out-dir)/tee.bin: $(link-out-dir)/tee-pager.bin \
 			 $(link-out-dir)/tee-init_mem_usage.txt \
 			./scripts/gen_hashed_bin.py
 	@$(cmd-echo-silent) '  GEN     $@'
-	$(q)./scripts/gen_hashed_bin.py \
+	$(q)load_addr=$(shell cat $(link-out-dir)/tee-init_load_addr.txt); \
```

Comment:
Right, and I suppose I will need to use pure shell also, not `$(shell ...)`. I'll check that. The advantage of `$(shell ...)` is that you see the address with `make V=1` but that's not that important.